Source/DayView.swift

```diff
@@ -60,7 +60,22 @@ public final class DayView: UIView, TimelinePagerViewDelegate {
     }
   }
 
-  public var calendar: Calendar = Calendar.autoupdatingCurrent
+  public var calendar: Calendar = Calendar.autoupdatingCurrent {
+    didSet {
+      // TODO: Should we observe calendar in state rather then change it manualy?
+      dayHeaderView.calendar = self.calendar
+      timelinePagerView.calendar = self.calendar
+
+      let date = self.state?.selectedDate ?? Date()
+      let newState = DayViewState(date: date, calendar: calendar)
+      newState.move(to: date)
+      state = newState
+      
+      // TODO: Shound we redraw dayView and its subvies? Or we should do this when calendar in state was changed?
+//      setNeedsDisplay()
```

Comment:
I think yes, as the 1st day of the week is based on the Calendar, which could change. So it there should be a call to `setNeedsDisplay`.
Also all of the subviews have to be redrawn.

Reply:
What about dayview's state? Should we redrawing subvies in response to a changes in state?

Reply:
I believe so, since that wouldn't happen too often. A change in state means a change in what should be displayed, so that should trigger a full relayout & redraw cycle.
I think, the best is to use `setNeedsLayout`, as the redraw will be triggered by the system when needed.